final_gen_solve/generateCSV.cpp

```diff
@@ -8,8 +8,8 @@ int main(int argc, char* argv[]){
     //Argument 2: Stops in Bx
     //Argument 3: Stops in Bz
     //Argument 4: Initial Bx
-    //Argument 5: Stop Bx
-    //Argument 6: Iitial Bz
+    //Argument 5: Initial Bz
+    //Argument 6: Stop Bx
     //Argument 7: Stop Bz;
     //Argument 8: file name
 
```

Comment:
Why is this needed? Why not do this in memory? It doesn't really make sense to have this extra step. Especially considering if `number_of_samples_to_generate` > `number_of_processes`, you could assign a subset of the samples to each node rank to utilize data parallelism